### PR TITLE
autocomplete

### PR DESCRIPTION
**Pull request by amcatko**

### CODE REVIEW

File: app/Presenters/templates/Homepage/templateHowDoI.latte [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                        </li>
+                        <li>
+                            <a href="#" class="more">... See all results >></a>
+                        </li>
+                    </ul>
+                    <div class="autocomplete__head">Not documented yet</div>
+                    <ul class="upvote__list">
+                        <li>
+                            <div class="upvote__item">
+                                <a href="#">
```
Comment (richard-ejem) [inline app/Presenters/templates/Homepage/templateHowDoI.latte]: @amcatko please don't do this as a link, only the upvote icon should be a link (so, add to some hover action and link only to the upvote icon)

File: app/Presenters/templates/Homepage/templateHowDoI.latte [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                                    <div class="votes">Votes: 123</div>
+                                </a>
+                            </div>
+                            <div class="upvote">
+                                <a href="#"><i class="glyphicon glyphicon-thumbs-up"></i>Upvote</a>
+                            </div>
+                        </li>
+                    </ul>
+                    <div class="autocomplete__head">Not found what you looked for?</div>
+                    <div><a href="#"  class="more">... Request documentation >></a></div>
```
Comment (richard-ejem) [inline app/Presenters/templates/Homepage/templateHowDoI.latte]: @amcatko 
* please remove double space before "class"
* replace >> with &gt;&gt; (or add some nicer glyphicon or fa-icon :) )

File: frontend/src/less/components/autocomplete.less [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            }
+        }
+    }
+}
+
+input.first:focus + .autocomplete.first,
+input.second:focus + .autocomplete.second,
+input.third:focus + .autocomplete.third {
+    display: block;
+}
```
Comment (richard-ejem) [inline frontend/src/less/components/autocomplete.less]: @amcatko please add newline to the end of file

File: frontend/src/less/components/autocomplete.less [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+            .glyphicon {
+                font-size: 20px;
+                line-height: 10px;
+                top: 5px;
+            }
+        }
+    }
+}
+
+input.first:focus + .autocomplete.first,
```
Comment (richard-ejem) [inline frontend/src/less/components/autocomplete.less]: @amcatko input.first is both too generic and too specific :)

- class "first", "second" etc is unnecessary - the + operator will do the work
- rather give some class to the input saying it has autocomplete, like input.with-autocomplete + .autocomplete

File: frontend/src/less/components/searchbox.less [hunk 5]
```diff
@@ -1,15 +1,44 @@
 #searchbox {
+    width: 563px;
+    max-width: 100%;
+    margin: 0 auto;
+    position: relative;
+    @media screen and (max-width: 800px) {
```
Comment (richard-ejem) [inline frontend/src/less/components/searchbox.less]: @amcatko please use bootstrap grid thresholds (@screen-??-max)

File: frontend/src/less/components/searchbox.less [hunk 6]
```diff
@@ -1,15 +1,44 @@
...
+        margin-top: 30px;
+    }
+    @media screen and (max-width: 600px) {
```
Comment (richard-ejem) [inline frontend/src/less/components/searchbox.less]: @amcatko please use bootstrap grid thresholds (@screen-??-max)

File: frontend/src/less/components/searchbox.less [hunk 7]
```diff
@@ -1,15 +1,44 @@
...
         line-height: 52px;
     }
 
+    >span {
+        position: absolute;
+        top: 15px;
+        left: -90px;
+        padding-right: 0px;
+        line-height: normal;
+        @media screen and (max-width: 800px) {
```
Comment (richard-ejem) [inline frontend/src/less/components/searchbox.less]: @amcatko please use bootstrap grid thresholds (@screen-??-max)

**Review by richard-ejem — CHANGES_REQUESTED** [review]

File: app/Presenters/templates/Homepage/templateHowDoI.latte (hunk 2 above)
Comment (amcatko) [reply app/Presenters/templates/Homepage/templateHowDoI.latte]: Fixed

File: app/Presenters/templates/Homepage/templateHowDoI.latte (hunk 1 above)
Comment (amcatko) [reply app/Presenters/templates/Homepage/templateHowDoI.latte]: Fixed

File: frontend/src/less/components/autocomplete.less (hunk 4 above)
Comment (amcatko) [reply frontend/src/less/components/autocomplete.less]: fixed

File: frontend/src/less/components/searchbox.less (hunk 5 above)
Comment (amcatko) [reply frontend/src/less/components/searchbox.less]: fixed

File: frontend/src/less/components/autocomplete.less (hunk 3 above)
Comment (amcatko) [reply frontend/src/less/components/autocomplete.less]: fixed

File: frontend/src/less/components/searchbox.less (hunk 6 above)
Comment (amcatko) [reply frontend/src/less/components/searchbox.less]: fixed

File: frontend/src/less/components/searchbox.less (hunk 7 above)
Comment (amcatko) [reply frontend/src/less/components/searchbox.less]: fixed

File: app/Presenters/templates/Homepage/templateHowDoI.latte [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+        <div class="center">
+            <p class="claim">Dev-stack solution based on Nette framework, which provides you with fully ready application skeleton
+                and environment <strong>to start developing a web application.</strong>
+            </p>
+            <pre>$ composer create-project instante/skeleton</pre>
+        </div>
+    </div>
+    <div class="blueline">
+        <div class="container center">
+            <div id="searchbox">
```
Comment (richard-ejem) [inline app/Presenters/templates/Homepage/templateHowDoI.latte]: There shouldn't be multiple elements of the same ID in a document, please use only CSS classes for styling.

File: app/Presenters/templates/Homepage/templateHowDoI.latte (hunk 1 above)
Comment (richard-ejem) [reply app/Presenters/templates/Homepage/templateHowDoI.latte]: The link is still present here :( just the upvote icon should be a link.

File: app/Presenters/templates/Homepage/templateHowDoI.latte [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+                <span>How do I</span>
+                <input type="text" name="search_how" placeholder="Create" class="with-autocomplete">
```
Comment (richard-ejem) [inline app/Presenters/templates/Homepage/templateHowDoI.latte]: Something went wrong with the magnifier icon: 
snimek obrazovky 2017-03-14 v 10 00 01

**Review by richard-ejem — CHANGES_REQUESTED** [review]

**Pull request merged**